Sources/Starscream/WebSocket.swift

```diff
@@ -57,6 +57,12 @@ public struct WSError: Error {
     public let code: Int
 }
 
+public enum WebSocketStatus {
```

Comment:
```suggestion
public enum WebSocketStatus: String {
```

How about using String instead of default Int?

Reply:
By default, Swift enums aren’t backed by anything. The default isn’t raw representable at all. 

With that in mind, do you still want this change?